index.bs

```diff
@@ -1324,43 +1332,31 @@ The [=sandboxed file system root=]'s [=query permission steps=] are the followin
 
 </div>
 
-## The {{FileSystemDirectoryHandle/getSystemDirectory()}} method ## {#api-getsystemdirectory}
+## The {{getSandboxedFileSystem()}} method ## {#api-getsandboxedfilesystem}
 
 <xmp class=idl>
-enum SystemDirectoryType {
-  "sandbox"
-};
-
-dictionary GetSystemDirectoryOptions {
-  required SystemDirectoryType type;
-};
-
 [SecureContext]
-partial interface FileSystemDirectoryHandle {
-  static Promise<FileSystemDirectoryHandle> getSystemDirectory(GetSystemDirectoryOptions options);
+partial interface Window {
+  Promise<FileSystemDirectoryHandle> getSandboxedFileSystem();
```

Comment:
Bikeshedding on the name a bit... 

* This is the only place "FileSystem" appears in the JS API, right? (not counting interface names). Should it just be "Directory" instead?  Behind the scenes we implement it as a separate namespace, but... from the perspective of this API which doesn't provide the same FS root like the legacy API, this is just a directory.

* Maybe "origin" instead of "sandboxed" so we're not throwing a new term into JS? Not sure about this... "getOriginDirectory()" seems a overbroad. "getPrivateDirectory()" ? "getOriginPrivateDirectory()" ? 

Sorry for not including this bikeshedding in the issue; I meant to LGTM the idea of a separate method, but discuss the name further. (no objections to landing the PR with the current name, though!)

Reply:
I like getOriginPrivateDirectory, so changed it to that for now.